### PR TITLE
Feature/delegate play

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

[this issue](https://github.com/HausDAO/daohaus-monorepo/issues/362)

## Changes

- new delegatingToMember, delegateOf, delegateOfCount fields
- shares token mapping updates to keep track of addresses a member is receiving delegation from 
- corresponding type and schema updates in dao-dat lib
- hooks up front end filters to delegate queries
- adds delegate tag to list view
 
## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

**Comment by skuhlmann** [comment]
some notes

{
  dao(id: "0xd55aba3a8ad548a5b1ced765530af0cadf8dbf83") {
    id
    sharesAddress
    lootAddress
    members {
      memberAddress
      shares
      delegatingTo
      delegateShares
      delegatingToMember {
        memberAddress
      }
      delegateOf {
        memberAddress
      }
      delegateOfCount
    }
  }

}
isDelegating
// delegatingTo !== memberAddress


is a delegate
// delegateOf has data


votePower
// change to use DelegateShares

delegateFilter
has delegate: {memberaddress: memberAddress, delegatingTo_not: memberAddress }
isDelegate: {memberaddress: memberAddress, delegateOfCount_not: "0" }
- 


how to exclude no share no loot and no delegate shares?

**Comment by skuhlmann** [comment]
@jordanlesich 

the sdk is doing some help to transform this data. you'll see a isDelegate boolean and delegatingTo that will have an address || be undefined.

ex:
```json
        {
          dao: '0x209866bbc39e0ac02ca7f2d0acd107ab95610439',
          name: null,
          safeAddress: '0x92201572155b316e8b435c00a2e6258da5b3a9ea',
          activeProposalCount: 0,
          totalProposalCount: '0',
          activeMemberCount: '3',
          votingPower: 33.3,
          networkId: '0x5',
          delegatingTo: undefined,
          isDelegate: true,
          memberAddress: '0xf100041473280b594d78ab5fa4c44ba81edd367b',
          contractType: 'Moloch V3',
          tokenBalances: [ [Object] ],
          fiatTotal: 0
        },
                {
          dao: '0xd55aba3a8ad548a5b1ced765530af0cadf8dbf83',
          name: 'Goerli Wurly 2',
          safeAddress: '0x234d77d86bcff4e965a9467a7ea67382df6e2e7d',
          activeProposalCount: 0,
          totalProposalCount: '0',
          activeMemberCount: '3',
          votingPower: 0,
          networkId: '0x5',
          delegatingTo: '0xae2f0af98a3fe6504321f00dc3f7dae0d2150ad0',
          isDelegate: false,
          memberAddress: '0xf100041473280b594d78ab5fa4c44ba81edd367b',
          contractType: 'Moloch V3',
          tokenBalances: [ [Object] ],
          fiatTotal: 0
        },



```

File: apps/hub-app/src/pages/HomePage.tsx [hunk 1]
```diff
@@ -85,11 +86,20 @@ const HomePage = () => {
       setLoading(true);
       try {
         const haus = temporaryInitHaus();
+
```
Comment (jordanlesich) [inline apps/hub-app/src/pages/HomePage.tsx:89]: imo, getting this query is a more separate concern than making the fetch (intention of this fn), I would refactor this into its own function. 

Would also remove variable assignment to remove extra moving parts

**Review by jordanlesich — APPROVED** [review]
Have the one note. Otherwise it looks good to me.

File: apps/hub-app/src/pages/HomePage.tsx (hunk 1 above)
Comment (jordanlesich) [reply apps/hub-app/src/pages/HomePage.tsx:89]: Just saw the delegate data. That's helpful. Thanks!

Comment (skuhlmann) [reply apps/hub-app/src/pages/HomePage.tsx:90]: @jordanlesich 

take a look at the refactor at your convenience.

**Pull request merged**